src/ScottPlot5/ScottPlot5/Style/Marker.cs

```diff
@@ -0,0 +1,33 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace ScottPlot.Style
+{
+    public enum MarkerShape
+    {
+        Circle
+    }
+
+    public struct Marker
```

Comment:
I think we should build-out Marker a little more. We can do it outside this PR if it gets complex though. 

Basically I think markers should be classes that implement `IMarker`. Marker classes could live in `ScottPlot.Markers` namespace, and `ScottPlot.Marker` would be a static class that gives quick access to common markers (kind of like colors). This would make it easier for users to add their own markers, and also for new marker shapes to be added without hunting around for switch statements in the code base. Something like:

```cs
public interface IMarker {
    void Color Color { get; set; }
    void double Size { get; set; }
    void Render(SKSurface surface, IEnumerable<Pixel> points);
}
```

...and I guess some markers would have additional fields like `LineWidth` that live in the concrete classes and not the interface? I'm open to suggestions, but this is the basic idea

Reply:
Yeah I like having an `IMarker`, though I don't mind an enum-based approach either. However, if we were to stick with enums I'd like if we had a central `DrawMarker` that handles it so we only have one switch statement. Kind of like `MarkerTools.DrawMarker`, though I believe that actually creates an `IMarker` which knows how to draw itself.

Reply:
I'm also recalling that `LegendItem` holds an `SKFont`, but I really don't think that's something we want to expose to the user. The `SKFont` class is pretty intense, and it also needs to be disposed.

Perhaps this can just be a `Font` struct with a `GetSKFont()` though. Not quite as much thought required as with markers.